faq.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: Frequently Asked Questions (FAQ)
+nav_order: 5
+---
+
+## General
+
+Q: What do we need to do to copy data to the DCC?
+
+A: Contact your center liaison so we understand your specific needs, upload your release-ready data with a descriptive top-level folder name, and iteratively annotate and validate your data as new metadata standards are released by the working groups with the [Data Curator App[(https://www.synapse.org/#!Wiki:syn20681266/ENTITY)] or the API. Here is the [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md) in detail.    
```

Comment:
The 'Data Ingress SOP' is linked a few times throughout this page, but the link appears to be broken?

Reply:
I think the links are actually meant to point to the docs 🤦🏻‍♀️ The SOP was v0, I believe -- I'm updating the links to their proper docs pages now.